.github/workflows/prune-images.yml

```diff
@@ -0,0 +1,63 @@
+name: Prune Images
+run-name: Prune old images from OpenShift
+
+on:
+  workflow_dispatch:
+    inputs:
+      applications:
+        description: 'Applications to prune'
+        required: true
+        default: 'web-sims, api-sims'
+      licensePlate:
+        description: 'License Plate'
+        required: true
+        default: '0c27fb'
+      environment:
```

Comment:
For clarification, the images are not environment-specific, what would be the intention to have the environment as an input parameter?
Maybe I am just missing how the script will work and I can hold those questions for the final PR 😉

Reply:
From my understanding on going through the script, the env is to get the DC deployed image, to get the list of images older than the DC deployed image for a given env.

Reply:
@dheepak-aot images are not environment-specific so we will not prune images per environment. We may use some env as PROD to figure out which ones can be removed, but it would not fits in the idea of prune per env.